zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -1596,7 +1597,7 @@ public void processPacket(ServerCnxn cnxn, ByteBuffer incomingBuffer) throws IOE
                         new ServerAuthenticationProvider.ServerObjs(this, cnxn),
                         authPacket.getAuth());
                 } catch (RuntimeException e) {
-                    LOG.warn("Caught runtime exception from AuthenticationProvider: {}", scheme, e);
+                    LOG.warn("Caught runtime exception from AuthenticationProvider: {} {}", scheme, e);
```

Comment:
Do we need to add an additional "{}" here? I don't think exception needs "{}".

Reply:
I thought it's parameterized logging but i'm not totally sure.